app/src/main/kotlin/com/wire/android/ui/common/LinkifyText.kt

```diff
@@ -215,19 +219,31 @@ private fun ClickableText(
     onLongClick: (() -> Unit)? = null,
 ) {
     val layoutResult = remember { mutableStateOf<TextLayoutResult?>(null) }
+    val tapped = remember { mutableStateOf(false) }
```

Comment:
Is it needed? I can see that it's being changed to `true` after starting handling `onPress` and then changed to `false` right after that, but the value itself isn't used anywhere 🤔